app/src/repositories/webtoon/webtoonRepository.js

```diff
@@ -1,33 +1,98 @@
 const db = require("../../config/db");
 
 class WebtoonRepository {
-  // 전체 웹툰 조회
-  async getAllWebtoons() {
-    const query = `
-    SELECT id, title, day_of_week, thumbnail_url
-    FROM webtoons;
-    `;
+  // 요일과 정렬조건을 기준으로 웹툰 데이터 조회
+  async getWebtoonsByDaySorted(day, sort) {
+    let query;
+    if (sort === "rating_avg") {
+      query = `
+      SELECT 
+        w.id,
+        w.title,
+        w.day_of_week,
+        w.thumbnail_url,
+        ROUND(AVG(e.rating_avg), 2) AS average_rating
+      FROM webtoons w
+      LEFT JOIN episodes e ON w.id = e.webtoon_id
+      where w.day_of_week = ?
```

Comment:
사소합니다만 컨벤션을 맞추면 좋을거같아요

```suggestion
      WHERE w.day_of_week = ?
```

Reply:
사소하지만 중요한 부분 짚어주셔서 감사합니다. 검토하면서 놓친 부분인데, 바로 반영하겠습니다!